### PR TITLE
Adding tox support to Reddwarf.

### PR DESCRIPTION
**Pull request by TimSimpsonR**
- Changed reddwarf.conf.test to reflect local mode.
- Updated fake keystone. Any token is treated as the tenant ID, and any tenant id starting with "admin" is treated as an admin.

### CODE REVIEW

**Comment by TimSimpsonR** [comment]
Requires:
https://github.com/hub-cap/python-reddwarfclient/pull/21
Tests:
https://github.com/hub-cap/reddwarf_lite-integration/pull/113

File: bin/reddwarf-manage [hunk 1]
```diff
@@ -86,6 +86,19 @@ class Commands(object):
...
+        from reddwarf.instance import models
+        from reddwarf.db.sqlalchemy import session
+        db_api.drop_db(self.conf)
+        self.db_sync(repo_path)
+        # Sets up database engine, so the next line will work...
+        session.configure_db(self.conf)
+        models.ServiceImage.create(service_name=service_name,
+                                   image_id=image_id)
+
+    _commands = ['db_sync', 'db_upgrade', 'db_downgrade', 'db_wipe']
```
Comment (cp16net) [inline bin/reddwarf-manage]: it looks like this is overriding line 87 above. Is that right?

Comment (TimSimpsonR) [inline bin/reddwarf-manage]: And getting rid of image_update to boot.  :(

Comment (TimSimpsonR) [inline bin/reddwarf-manage]: Fixed.

**Comment by cp16net** [comment]
+1

File: reddwarf/tests/fakes/keystone.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        return 'HTTP_%s' % key.replace('-', '_').upper()
+
+    def _add_headers(self, env, headers):
+        """Add http headers to environment."""
+        for (k, v) in headers.iteritems():
+            env_key = self._header_to_env_var(k)
+            env[env_key] = v
+
+    def get_admin_token(self):
+        return "ABCDEF0123456789"
```
Comment (ed-) [inline reddwarf/tests/fakes/keystone.py:53]: That's the same as the combination on my luggage!

**Comment by ed-** [comment]
+1

**Comment by jcru** [comment]
+1

**Comment by imsplitbit** [comment]
+1

**Pull request merged**